src/screens/Technology/components/Details/index.tsx

```diff
@@ -101,64 +106,190 @@ const Stages = ({ currentStep }: StagesProps) => (
-    </S.Description>
-    <S.Description>
-      <S.Highlight>Duração do processo de instalação da tecnologia: </S.Highlight>
-      5 dias.
-    </S.Description>
-  </S.AccordionItemWrapper>
-);
+  return (
+    <S.AccordionItemWrapper>
+      <S.Subtitle>Identificação</S.Subtitle>
+      <S.DetailsText>
+        <S.Highlight>Título: </S.Highlight>
+        {technology.title}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Categoria: </S.Highlight>
+        {technology.taxonomies?.category}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Classificação: </S.Highlight>
+        {technology.taxonomies?.classification}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Dimensão: </S.Highlight>
+        {technology.taxonomies?.dimension}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Público-alvo: </S.Highlight>
+        {technology.taxonomies?.target_audience}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Bioma: </S.Highlight>
+        {technology.taxonomies?.biome}
+      </S.DetailsText>
+      <S.Subtitle>Estágio de desenvolvimento</S.Subtitle>
+      <Stages currentStep={technology.currentLevel || 1} />
+    </S.AccordionItemWrapper>
+  );
+};
+
+export const Characteristics = () => {
+  const technology = useTechnology();
+
+  return (
+    <S.AccordionItemWrapper>
+      <S.Subtitle>Objetivos</S.Subtitle>
+      <S.DetailsText>
+        <S.Highlight>Objetivo Principal: </S.Highlight>
+        {technology.primary_purpose}
+      </S.DetailsText>
+      <S.Subtitle>Aplicação</S.Subtitle>
+      <S.DetailsText>
+        <S.Highlight>Onde é a Aplicação: </S.Highlight>
+        {technology.application_mode}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Pré-requisitos para a implantação: </S.Highlight>
+        {technology.requirements}
+      </S.DetailsText>
+      <S.DetailsText>
+        <S.Highlight>Duração do processo de instalação da tecnologia: </S.Highlight>
+        {`${technology.installation_time} dias.`}
+      </S.DetailsText>
+    </S.AccordionItemWrapper>
+  );
+};
+
+interface CostsItemsProps {
+  data: {
+    id: number,
+    description: string,
+    type: string,
+    quantity: number,
+    value: number,
+    measure_unit: string
+  }[]
+}
+
+export const CostsItems = ({ data } : CostsItemsProps) => {
+  const emptyMessage = 'Nenhum custo cadastrado.';
+  let isEmpty = false;
+
+  if (!data?.length || !Array.isArray(data)) {
+    isEmpty = true;
+  }
+
+  const typesValues = [
+    {
+      value: 'service',
+      label: 'Serviço',
```

Comment:
It might be a good idea to extract it to a util/helper file to be more reusable

Reply:
Done!